src/styles/_config.example.scss

```diff
@@ -1,9 +0,0 @@
-/**
```

Comment:
Is there a reason for deleting this file?

Reply:
whoops